.circleci/config.yml

```diff
@@ -353,7 +353,6 @@ workflows:
                 - quay.io/astronomer/ap-auth-sidecar:1.27.1
                 - quay.io/astronomer/ap-awsesproxy:1.5.0-11
                 - quay.io/astronomer/ap-base:3.18.9
-                - quay.io/astronomer/ap-blackbox-exporter:0.25.0-3
```

Comment:
why did we remove this?
Double entry?

Reply:
@rishkarajgi it looks like this was removed due to pre-commit run, the pre-commit uses bin/show-docker-images.py script which is currently unable to list quay.io/astronomer/ap-blackbox-exporter image due to which this has been removed, I will try to check this and update.

Reply:
global.blackboxExporterEnabled we already have this flag is that not working

Reply:
@pgvishnuram it is commented, LEt me use that only.